project/project-info.conf

```
@@ -112,4 +112,14 @@ project-info {
       }
     ]
   }
+  loglevels-log4j2: ${project-info.shared-info} {
+    title: "Dynamic Log Levels"
+    levels: [
+      { // keep in sync with README.md
+        readiness: Supported
```

Not that it is much code to support, but let's start with:
```suggestion
        readiness: CommunityDriven
```